db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -360,9 +360,27 @@ class NonBatchedOpsStressTest : public StressTest {
       }
       // found case
       thread->stats.AddGets(1, 1);
+      if (thread->shared->Get(rand_column_families[0], rand_keys[0]) ==
```

Comment:
What if `read_opts_copy.timestamp` is set by `MaybeUseOlderTimestampForPointLookup`?

Reply:
Since we only have the latest expected state, I think we can only do the verification when timestamp is not set. I'll update accordingly.